siriuspy/siriuspy/devices/blm.py

```diff
@@ -9,17 +10,70 @@ class BLM(_Device):
 
     class DEVICES:
         """Devices names."""
-        SENSOR1 = _BLMSearch.conv_blmname_2_devname('SENSOR1')
-        SENSOR2 = _BLMSearch.conv_blmname_2_devname('SENSOR2')
-        SENSOR3 = _BLMSearch.conv_blmname_2_devname('SENSOR3')
-        SENSOR4 = _BLMSearch.conv_blmname_2_devname('SENSOR4')
-        SENSOR5 = _BLMSearch.conv_blmname_2_devname('SENSOR5')
-        SENSOR6 = _BLMSearch.conv_blmname_2_devname('SENSOR6')
-        SENSOR7 = _BLMSearch.conv_blmname_2_devname('SENSOR7')
-        SENSOR8 = _BLMSearch.conv_blmname_2_devname('SENSOR8')
-        ALL = (
-            SENSOR1, SENSOR2, SENSOR3, SENSOR4,
-            SENSOR5, SENSOR6, SENSOR7, SENSOR8)
+        __SENSOR1, __SENSOR2, __SENSOR3, __SENSOR4 = 4 * (None, )
```

Comment:
just as a suggestion, python also allows:
```python3
__SENSOR1 = __SENSOR2 = __SENSOR3 = __SENSOR4 = None
__SENSOR5 = __SENSOR6 = __SENSOR7 = __SENSOR8 = None
```

Reply:
cleaner. I forgot about it. implemented